paimon-web-ui-new/src/views/cdc/components/list/index.tsx

```diff
@@ -247,12 +35,24 @@ export default defineComponent({
   render() {
     return (
       <div class={styles['list-page']}>
-        <n-data-table
-          columns={this.columns}
-          data={this.data}
-          pagination={this.pagination}
-          bordered={false}
-        />
+         <NSpace vertical>
+            <NDataTable
+              columns={this.columns}
+              data={this.data}
+            />
+            <NSpace justify='center'>
+              <NPagination
+                v-model:page={this.pagination.page}
+                v-model:page-size={this.pagination.pageSize}
+                item-count={this.pagination.count}
+                show-size-picker
+                page-sizes={this.pagination.pageSizes}
+                show-quick-jumper
+                onUpdatePage={this.pagination.onChange}
+                onUpdatePageSize={this.pagination.onUpdatePageSize}
+              />
+            </NSpace>
+          </NSpace>
```

Comment:
These components can be automatically imported by `n-space` and do not need to be imported by yourself.